scripts/test_perf.py

```diff
@@ -5,10 +5,15 @@
 from datetime import datetime, timedelta
```

Comment:
should probably add this to the dockerignore. @yorickvP, we made cognix also respect .dockerignore, right?